openslides_backend/models/checker.py

```diff
@@ -562,6 +564,21 @@ def get_enum_from_collection_field(
         field_type = self.get_model(collection).get_field(field)
         return field_type.constraints.get("enum")
 
+    def check_amendment_paragraph(self, model: Dict[str, Any], collection: str) -> None:
```

Comment:
Can you rename this to something like `check_special_fields` and move the recommendation extension validation code from https://github.com/OpenSlides/openslides-backend/blob/d30f2a82f49176442b8ebc1c0ce15df8c87b414f/openslides_backend/models/checker.py#L684-L705 also in here to unify the handling of such fields with extra validation?

Reply:
Done.